core/src/main/java/org/elasticsearch/search/aggregations/metrics/sum/Sum.java

```diff
@@ -29,4 +33,13 @@
      * The sum.
      */
     double getValue();
+
+    // TODO we could pull common xContent rendering into a static method in the interface
```

Comment:
I think we should do it only when it's very simple to do. It is a bit funky to have this static method in the interface but that is the best place for it. @tlrx what do you think?